plugins/plugin-docker/che-plugin-docker-client/pom.xml

```diff
@@ -23,18 +23,14 @@
     <name>Che Plugin :: Docker :: Docker Client</name>
     <properties>
         <findbugs.failonerror>false</findbugs.failonerror>
-        <com.openshift.openshift-restclient-java.version>5.2.0-SNAPSHOT</com.openshift.openshift-restclient-java.version>
+        <io.fabric8.openshift-client.version>1.4.26</io.fabric8.openshift-client.version>
     </properties>
     <dependencyManagement>
         <dependencies>
             <dependency>
-                <groupId>com.openshift</groupId>
-                <artifactId>openshift-restclient-java</artifactId>
-                <version>${com.openshift.openshift-restclient-java.version}</version>
-                <exclusions>
-                  <exclusion>
-                  </exclusion>
-                </exclusions>
+                <groupId>io.fabric8</groupId>
+                <artifactId>openshift-client</artifactId>
+                <version>${io.fabric8.openshift-client.version}</version>
```

Comment:
<dependencyManagement> block should be in https://github.com/eclipse/che-dependencies/blob/master/pom.xml

and each dependency needs to have a CQ
I see one (your request) which is https://dev.eclipse.org/ipzilla/show_bug.cgi?id=12209 but it's for version 1.4 while here I see 1.4.26